src/tasks/MonoTargetsTasks/ILStrip/AssemblyStripper/AssemblyStripper.cs

```diff
@@ -192,6 +203,15 @@ void PatchResources()
             }
         }
 
+        void SortCustomAttributes()
```

Comment:
can you please add a comment about why we need to sort the custom attributes here?